pkg/controller/operators/catalog/operator.go

```diff
@@ -1578,9 +1578,9 @@ func (o *Operator) ExecutePlan(plan *v1alpha1.InstallPlan) error {
 		}
 
 		switch step.Status {
-		case v1alpha1.StepStatusPresent, v1alpha1.StepStatusCreated, v1alpha1.StepStatusWaitingForAPI:
+		case v1alpha1.StepStatusPresent, v1alpha1.StepStatusCreated:
```

Comment:
To make sure that I understand this change, if ` v1alpha1.StepStatusWaitingForAPI` was here, OLM would just skip the step ,correct?

Reply:
Yes